### PR TITLE
Upgrade tonic to v0.11

### PR DESCRIPTION
**Pull request by fenollp**
And other (not super related nor important) things :)

### CODE REVIEW

**Comment by fenollp** [comment]
@mechiru Ping just to make sure you'll be notified. 
I'm open to chat via email BTW if that can make our communication easier :)

File: README.md [hunk 1]
```diff
@@ -58,6 +58,7 @@ The list of available features can be found [here](./google-api-proto/Cargo.toml
 | 1.243.0 <=       | 0.8.x  | 0.8.x       |
 | 1.280.0 <=       | 0.9.x  | 0.9.x       |
 | 1.415.0 <=       | 0.10.x | 0.10.x      |
+| 1.512.0 <=       | 0.11.x | 0.11.x      |
```
Comment (mechiru) [inline README.md]:
```suggestion
| 1.515.0 <=       | 0.11.x | 0.11.x      |
```

**Review by mechiru — CHANGES_REQUESTED** [review]
@fenollp Thank you for your contribution!
Code change looks good.
Merge once you have corrected the tonic support version of the document.

File: README.md (hunk 1 above)
Comment (mechiru) [reply README.md]: If you are willing to modify it here, we will respond. Please mentions it to me.

**Review by mechiru — APPROVED** [review]
Thanks 😉

**Comment by fenollp** [comment]
Ah it looks like `515` was pushed to crates.io https://crates.io/crates/google-api-proto/1.515.0/dependencies but
* this is the version point we pinned to use tonic `0.11` but it appears as requiring `0.10`
* the corresponding tag does not show on https://github.com/mechiru/google-api-proto/tags

Is your depoy pipeline broken? (and thanks for your time)

**Pull request merged**